source/acquaman/AMEXAFSScanActionControllerAssembler.cpp

```diff
@@ -213,10 +213,12 @@ AMAction3 *AMEXAFSScanActionControllerAssembler::generateActionTreeForEXAFSStepA
 
 			for(int x = 0; x < detectors_->count(); x++){
 
-				AMAction3 *detectorSetDwellAction = detectors_->at(x)->createSetAcquisitionTimeAction(variableIntegrationTimes.at(i));
+				if(detectors_->at(x)->supportsSynchronizedDwell()) {
```

Comment:
Just want to confirm that this won't affect the usage of some detectors. I noticed that the `VESPERSCCDDetector/VESPERSFourElementVotexDetector/VESPERSSingleElementVotexDetector/SXRMBFourElementVortexDetector/SXRMBBrukerDetector/REIXSESMCPDetector/IDeasKETEKDetector/IDEAS13ElementGeDetector/AMXspress3XRFDetector(super class of n-element detector of BioXAS)` doesn't support sychronized dwell.